STEP/ArduinoCompiler.cs

```diff
@@ -71,12 +77,26 @@ public void Upload(string filename)
         ArduinoSketchUploader uploader = new ArduinoSketchUploader(new ArduinoSketchUploaderOptions()
         {
             FileName = directoryPath + $"/{filename}/build/arduino.avr.uno/{filename}.ino.hex",
-            PortName = "COM3", // Can be omitted to try to auto-detect the COM port.
+            PortName = _port, // Can be omitted to try to auto-detect the COM port.
```

Comment:
Kan man undlade at inkludere porten når man kører compileren, og så prøver den at auto-sætte den?
Hvis det er muligt vil det give lidt mere brugervenlighed, og hvis den ikke finder den rigtige port automatisk, så har man mulighed for at sætte den manuelt

Reply:
Jeg fikser det imorgen 👍